### ISSUE TITLE
Error 500 after local.yaml is created

### ISSUE THREAD

**Issue by codexdb** [kevinpapst/kimai2#1204]
**Describe the bug**
After creating the file local.yaml with content as proposed in 
https://www.kimai.org/documentation/configurations.html
I cleared cache, made warmup and restarted apache2 server.
Tried to login, server gave error 500.

**Logfile**
```
[2019-10-29 14:53:26] request.INFO: Matched route "fos_user_security_login". {"route":"fos_user_security_login","route_parameters":{"_route":"fos_user_security_login","_controller":"fos_user.security.controller:loginAction","_locale":"de"},"request_uri":"https://time.codexdb.com/de/login","method":"GET"} []
[2019-10-29 14:53:26] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"secured_area","authenticators":1} []
[2019-10-29 14:53:26] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"secured_area","authenticator":"App\\Security\\TokenAuthenticator"} []
[2019-10-29 14:53:26] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"secured_area","authenticator":"App\\Security\\TokenAuthenticator"} []
[2019-10-29 14:53:26] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2019-10-29 14:53:26] cache.WARNING: Failed to save key "%5B8dd877ae0af5cd301970284f8bcfe8b2%5D%5B1%5D" of type Doctrine\ORM\Query\ParserResult: file_put_contents(/var/www/clients/client1/web11/web/kimai2/var/cache/prod/pools/kWxkx0uShF/5db84456711463.91619213): failed to open stream: Permission denied {"key":"%5B8dd877ae0af5cd301970284f8bcfe8b2%5D%5B1%5D","exception":"[object] (ErrorException(code: 0): file_put_contents(/var/www/clients/client1/web11/web/kimai2/var/cache/prod/pools/kWxkx0uShF/5db84456711463.91619213): failed to open stream: Permission denied at /var/www/clients/client1/web11/web/kimai2/vendor/symfony/cache/Traits/FilesystemCommonTrait.php:92)"} []
[2019-10-29 14:53:26] request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "An exception has been thrown during the rendering of a template ("Failed to create "/var/www/clients/client1/web11/web/kimai2/var/cache/prod/translations": mkdir(): Permission denied.")." at /var/www/clients/client1/web11/web/kimai2/templates/bundles/FOSUserBundle/Security/login.html.twig line 4 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): An exception has been thrown during the rendering of a template (\"Failed to create \"/var/www/clients/client1/web11/web/kimai2/var/cache/prod/translations\": mkdir(): Permission denied.\"). at /var/www/clients/client1/web11/web/kimai2/templates/bundles/FOSUserBundle/Security/login.html.twig:4, Symfony\\Component\\Filesystem\\Exception\\IOException(code: 0): Failed to create \"/var/www/clients/client1/web11/web/kimai2/var/cache/prod/translations\": mkdir(): Permission denied. at /var/www/clients/client1/web11/web/kimai2/vendor/symfony/filesystem/Filesystem.php:105)"} []
```

**Desktop/Smartphone**
 - Device: Linux server
 - OS: debian buster
 - Browser: Tested on chrome, firefox, edge
 - Version  current versions

**Additional context**
 - Kimai version: 1.4.2
 - PHP version: 7.3

As you can see, the owner of the kimai folder is web11 in group client1.
To clear error message I tried to

- clear the content of local.layout -> no success
- changed permissions of kimai folder to 777 -> no success
- removed local.yaml -> success

After each trial I cleared cache, made warmup and restarted apache2.
The first two cases displayed always in web-developer window an error
in different javascript files so that the layout could not be loaded completely.

I hope that the information can give you an idea of the problem. 
Thanks in advanced.

**Comment by kevinpapst** [comment]
What is local.layout?
The error clearly says: `file_put_contents(/var/www/clients/client1/web11/web/kimai2/var/cache/prod/pools/kWxkx0uShF/5db84456711463.91619213): failed to open stream: Permission denied  `
You need to apply the correct permissions, see https://www.kimai.org/documentation/installation.html

**Comment by codexdb** [comment]
Sorry this is only a typo in description. Naturally I called the 
file local.yaml.
Owner and group of the folder are made by a control panel called
ispconfig. Following your proposition and changing the group from
client1 to www-data had no effect.

I think that the main problem is the creation of local.yaml (I needed
for changing company and logo). Before that everything worked
perfectly (so I have to admire that you made a perfect job).
Suddenly, after controlling the permissions for the var folder I found a
change in ownership for the subfolder prod from web11 to root and 
fom rights 777 to 755. Could this be helpful?

Another thanks for your efforts.
